canlogserver.c

```diff
@@ -102,7 +103,12 @@ void print_usage(char *prg)
 	fprintf(stderr, "\n");
 	fprintf(stderr, "When using more than one CAN interface the options\n");
 	fprintf(stderr, "m/v/i/e have comma separated values e.g. '-m 0,7FF,0'\n");
-	fprintf(stderr, "\nUse interface name '%s' to receive from all CAN interfaces.\n\n", ANYDEV);
+	fprintf(stderr, "\nUse interface name '%s' to receive from all CAN interfaces.\n", ANYDEV);
+	fprintf(stderr, "\n");
+	fprintf(stderr, "After running canlogserver, connect to it via TCP to get logged data\n");
+	fprintf(stderr, "ex)\n");
```

Comment:
is "ex)" a common pattern?

what about "e.g. with 'nc localhost %d'\n", DEFPORT);